jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/util/RSSUtils.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (C) 2011  JTalks.org Team
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+package org.jtalks.jcommune.web.util;
+
+/**
+ * @author denis.berezhnoy
+ */
+public class RSSUtils {
+    /**
+     * Method for skip invalid characters in content for RSS feed.
+     *
+     * @param in - post content.
+     */
+    public static String skipInValidXMLChars(String in) {
+        StringBuffer out = new StringBuffer();
+        char current;
+
+        if (in == null || ("".equals(in))) return "";
+        for (int i = 0; i < in.length(); i++) {
+            current = in.charAt(i);
+            if ((current == 0x9) ||
```

Comment:
These should be constants. Also you should explain why these chars are invalid for XML. And it seems to me that you're just skipping the characters.. That's an important decision that should be carefully documented and justified.